src/Groups/MatrixDisplay.jl

```diff
@@ -108,7 +118,7 @@ The following attributes of `io` are supported.
   and to create portions according to the screen width otherwise,
 
 ## Examples
-```jldoctest
+```jldoctest; setup = :(Oscar.allow_unicode(true))
```

Comment:
Will this turn off the unicode support later on again? If not, couldn't this affect other examples?

Reply:
In fact I had expected all kinds of technical problems, the current test failures for Julia nightly are perhaps some of them.
(The failures happen in `printing.jl`, the first test file that gets evaluated.)
I am now also checking the behaviour of the `setup` code w.r.t. different doctests

Reply:
@fingolfin I have changed the approach: Do not set globals in `setup` code of jldoctest.

Thus setting/resetting the `is_unicode_allowed` flag is now visible in the docstring examples.

Still it might be a bad idea to show unicode in the manual, whereas it is switched off in the session by default.
On the other hand, unicode looks just nicer, in particular in the manual.
(Perhaps we will eventually need a general decision that unicode is banned also from the manual.)